src/components/vm/nics/vmNicsCard.tsx

```diff
@@ -664,16 +677,65 @@ export class VmNetworkTab extends React.Component<VmNetworkTabProps, VmNetworkTa
                 let column = null;
                 if (typeof d.value === 'string') {
                     if (target[d.value] !== undefined) {
-                        column = { title: <div id={`${id}-network-${networkId}-${d.value}`}>{String(target[d.value])}</div>, props: d.props };
+                        column = { title: <div id={`${id}-network-${networkId}-${d.value}`}>{String(target[d.value])}</div> };
                     }
                 }
                 if (typeof d.value === 'function') {
-                    column = { title: d.value(target, networkId), props: d.props };
+                    column = { title: d.value(target, networkId) };
                 }
                 return column;
             });
+
+            const Description = ({ term, children } : { term: string, children: React.ReactNode }) => (
+                <DescriptionListGroup>
+                    <DescriptionListTerm>{term}</DescriptionListTerm>
+                    <DescriptionListDescription>{children}</DescriptionListDescription>
+                </DescriptionListGroup>
+            );
+
+            const expandedContent = (
+                <DescriptionList isHorizontal>
+                    <Description term={_("Model type")}>
+                        <Flex
+                            spaceItems={{ default: 'spaceItemsSm' }}
+                            alignItems={{ default: 'alignItemsCenter' }}
+                            id={`${id}-network-${networkId}-model`}
+                        >
+                            <FlexItem>{target.model}</FlexItem>
+                            {needsShutdownIfaceModel(vm, target) &&
+                                <NeedsShutdownTooltip
+                                    iconId={`${id}-network-${networkId}-model-tooltip`}
+                                    tooltipId="tip-network"
+                                />
+                            }
+                        </Flex>
+                    </Description>
+                    <IPDescriptions
+                        ipInterfaces={this.state.ips.filter(ip => ip.mac === target.mac)}
+                        id={id}
+                        networkId={networkId}
+                    />
+                    <NetworkSourceDescriptions
+                        network={target}
+                        networkId={networkId}
+                        vm={vm}
+                        hostDevices={this.hostDevices}
+                    />
+                </DescriptionList>
+            );
+
+            const rowId = networkId;
+
             networkId++;
-            return { columns, props: { key: cockpit.format("$0-$1-$2", target.mac, target.address.bus || networkId, target.address.slot || '') } };
+            return {
+                columns,
+                props: {
+                    key: cockpit.format("$0-$1-$2", target.mac, target.address.bus || networkId, target.address.slot || ''),
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2324-d14228b1-20250918-114501-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)